data/data/baremetal/main.tf

```diff
@@ -20,7 +20,6 @@ module "bootstrap" {
 module "masters" {
   source = "./masters"
 
-  ironic_uri     = var.ironic_uri
```

Comment:
Why is this being removed?

Reply:
See https://github.com/openshift-metal3/kni-installer/pull/137/commits/ea152901d6a40052520727713078d2d05ac0e4de

```
baremetal: remove unused ironic_uri var from masters

This was used for the now deleted wait-for-ironic stuff.
```

Reply:
Sorry, I read this entirely wrong, didn't realize it was under masters.  Thanks.